CameraOverlay/research_and_development.py

```diff
@@ -0,0 +1,51 @@
+from PIL import Image, ImageDraw, ImageFont
```

Comment:
What is this file used for? Just as a test?

Reply:
to generate the static images. i thought that would be helpful to generate images on laptops and see if the format is as per the rider's expectations :)